mailpoet/assets/js/src/newsletter_editor/blocks/coupon.ts

```diff
@@ -8,6 +8,7 @@ import _ from 'underscore';
 import jQuery from 'jquery';
 import 'backbone.marionette';
 import { MailPoet } from '../../mailpoet';
+import 'select2';
```

Comment:
Question: Do we need to import `select2` here?

I can see we used it without import [here](https://github.com/mailpoet/mailpoet/blob/d00195a239f29342fa505972735a49f08e749626/mailpoet/assets/js/src/newsletter_editor/blocks/automatedLatestContent.js#L268-L269)

Reply:
I'm not sure here. Maybe it was imported by IDE, and maybe I added it because it's in `abandonedCartContent.js` and `products.js`. I can remove it.